clusters.cnp.example.org/.crossplane/allocatedcluster.ui-schema.yaml

```diff
@@ -0,0 +1,83 @@
+initialConfig: true
+configSections:
+- title: CNP Kubernetes Cluster
+  description: Provide the following details to provision a Kubernetes cluster
+  items:
+  - name: region
+    controlType: singleSelect
+    path: .spec.region
+    title: Region
+    description: Region of the resources that will be deployed.
+    enum:
+    - 'ap-east-1'
+    - 'ap-northeast-1'
+    - 'ap-northeast-2'
+    - 'ap-south-1'
+    - 'ap-southeast-1'
+    - 'ap-southeast-2'
+    - 'ca-central-1'
+    - 'eu-central-1'
+    - 'eu-north-1'
+    - 'eu-west-1'
+    - 'eu-west-2'
+    - 'eu-west-3'
+    - 'me-south-1'
+    - 'sa-east-1'
+    - 'us-east-1'
+    - 'us-east-2'
+    - 'us-west-2'
+    validation:
+      - required: true
+- title: Generate Provider Secret
```

Comment:
@negz If CNP `Cluster` will be published so they can be created on demand, then we probably don't want `Provider` and the provider `Secret` to be created as part of creating an instance of a `Cluster`. Suggest removing this for now and having the `Provider` and provider `Secret` be manually created for now until we have a better longer term solution for expressing `Provider` dependencies in child `Configuration` packages.
/cc @lukeweber @rbwsam @rathpc